Sources/KSCrashRecording/KSCrashReportC.c

```diff
@@ -904,7 +905,9 @@ static void writeNotableStackContents(const KSCrashReportWriter *const writer,
     char nameBuffer[40];
     for (uintptr_t address = lowAddress; address < highAddress; address += sizeof(address)) {
         if (ksmem_copySafely((void *)address, &contentsAsPointer, sizeof(contentsAsPointer))) {
-            sprintf(nameBuffer, "stack@%p", (void *)address);
+            memcpy(nameBuffer, "stack@0x", 8);
+            char *addressStart = nameBuffer + 8;
```

Comment:
Can we use `sizeeof` to make it more error-proof?


```suggestion
            static const char prefix[] = "stack@0x";
            memcpy(nameBuffer, prefix, sizeof(prefix) - 1);
            char *addressStart = nameBuffer + sizeof(prefix) - 1;
```